run.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-./bin/Debug/raytracing.exe --output=test.ppm --width=200 --height=200 --samples=64
\ No newline at end of file
+./bin/Debug/raytracing.exe --output=test.ppm --width=800 --height=800 --samples=32
```

Comment:
Needed to update the run.sh to test a proper full-resolution image.

Reply:
Out of curiosity, how long it takes?